src/containers/SearchPage/SearchPageWithMap.js

```diff
@@ -411,11 +429,16 @@ export class SearchPageComponent extends Component {
 
     // Selected aka active secondary filters
     const selectedSecondaryFilters = hasSecondaryFilters
-      ? validFilterParams(validQueryParams, {
-          listingFieldsConfig: customSecondaryFilters,
-          defaultFiltersConfig: [],
-          listingCategories,
-        })
+      ? validFilterParams(
+          validQueryParams,
+          {
+            listingFieldsConfig: customSecondaryFilters,
+            defaultFiltersConfig: [],
+            listingCategories,
+            activeListingTypes,
+          },
+          currentPathParams
+        )
```

Comment:
This feels weird to me. There are 2 different parameters, but they are not next to each other. 
`validFilterParams(validQueryParams, currentPathParams, filterConfigs)`
`validFilterParams(validQueryParams, currentPathParams, filterConfigs, options)`


This could perhaps even keep the original signature:
```suggestion
      ? validFilterParams(
          {
            ...validQueryParams,
            ...(listingTypePathParam ? { pub_listingType: listingTypePathParam } : {}),
          },
          {
            listingFieldsConfig: customSecondaryFilters,
            defaultFiltersConfig: [],
            listingCategories,
            activeListingTypes,
          })
```

Note: you have used the same concept inside **_omitLimitedListingFieldParams_** (SearchPage.shared.js):
![Screenshot 2025-05-08 at 12 26 17](https://github.com/user-attachments/assets/33bd947b-f525-41f3-ab5f-6999662a1887)

Reply:
Had an IRL conversation about this, and we decided to pass currentPathParams inside filterConfig